### PR TITLE
New pages

### PR DESCRIPTION
**Pull request by mikkelcodes**
To do:
- Convert temp inline styles to tailwind or css modules
- Create stories
- Refactor out components from index and about page
- Responsive updates
- Slider functionality
- Videos

### CODE REVIEW

File: packages/website/src/components/hero/index.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+export const Hero = () => (
+  <div className="bg-navy w-full text-6xl flex justify-center items-center h-screen -mt-25 p-12 relative">
+    <Title bold={false} size="text-6xl leading-tight">
+      <div>Vi bygger </div>
+      <div className="font-black">Norges mest attraktive</div>
+      <span>konsulentselskap</span>
+    </Title>
+    <div className="sm:h-3 h-4" />
+    <div className="absolute text-white right-0 bottom-0 text-base mr-5 mb-40 transform -rotate-90 tracking-wider font-thin flex">
```
Comment (TheKnarf) [inline packages/website/src/components/hero/index.js:13]: This right-aligned `scroll to continue` could probably be its own component.

Comment (mikkelcodes) [reply packages/website/src/components/hero/index.js:13]: True

File: packages/website/src/components/icon/scroll_to_continue_arrow.svg [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="17.661" height="99.998" viewBox="0 0 17.661 99.998">
```
Comment (TheKnarf) [inline packages/website/src/components/icon/scroll_to_continue_arrow.svg:1]: Is this in use? Or is it just a duplicate of `<Icon.ScrollArrow />`?

File: packages/website/src/pages/om-oss.js [hunk 3]
```diff
@@ -23,20 +25,20 @@ const About = () => {
             <div className="w-full">
               <Image
                 fluid={data.aboutUsTop.childImageSharp.fluid}
-                style={{ height: '600px' }}
+                style={{ height: '600px', width: '100%' }}
```
Comment (TheKnarf) [inline packages/website/src/pages/om-oss.js:28]: Refactor into a className instead of using the `style`-propertie

**Review by TheKnarf — APPROVED** [review]

**Pull request merged**